recipes/catch2/2.x.x/test_package/CMakeLists.txt

```diff
@@ -16,3 +16,13 @@ if(WITH_MAIN)
   add_executable(standalone 200-standalone.cpp)
   target_link_libraries(standalone PRIVATE Catch2::Catch2WithMain)
 endif()
+
+if(NOT WITH_BENCHMARK)
+  add_definitions(-DADD_MAIN)
+endif()
+add_executable(benchmark 300-benchmark.cpp)
```

Comment:
Why is the benchmark executable always built?
Doesn't it belong in a if block?

Reply:
Benchmark tests could be built by adding adding the `CATCH_CONFIG_MAIN` macro in those cases. I have removed all that now and just testing the case when `enable_benchmark` is True